src/NuGet.Core/NuGet.Protocol.Core.v3/Utility/ProtocolUtility.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Net;
+using NuGet.Common;
+
+namespace NuGet.Protocol
+{
+    public class ProtocolUtility
```

Comment:
Good catch, it should have been. This will get moved over to NetworkProtocolUtility